src/main/java/uk/nhs/digital/nhsconnect/nhais/service/edifact_to_fhir/AcceptanceTransactionMapper.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Interchange;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionType;
+
+@Component
+public class AcceptanceTransactionMapper implements TransactionMapper {
```

Comment:
AcceptanceTransactionMapper not used?

Reply:
sorry, it's 'TransactionMapper'. 
can you add unit tests?

Reply:
will be added by @grzegorz-robak  in his PR

Reply:
I'm doing approval right now